src/pca9685.rs

```diff
@@ -86,16 +86,20 @@ impl Pca9685DeviceBuilder {
     /// Builds the `Pca9685Device`.
     pub fn build(self) -> Result<Pca9685Device, Box<dyn Error>> {
         let device = I2cdev::new(self.i2c_bus)?;
-        let pwm = Pca9685::new(device, self.address).expect("Failed to open PWM controller");
+        let mut pwm = Pca9685::new(device, self.address).expect("Failed to open PWM controller");
 
         let oe_pin = {
             let pin = Pin::new(self.oe_pin_number);
             pin.export()?;
             sleep(Duration::from_millis(30));
-            pin.set_direction(Direction::Low)?;
+            pin.set_direction(Direction::High)?;
```

Comment:
I was wondering what a low/high direction even means, here it is:

>	"direction" ... reads as either "in" or "out". This value may
		normally be written. Writing as "out" defaults to
		initializing the value as low. To ensure glitch free
		operation, values "low" and "high" may be written to
		configure the GPIO as an output with that initial value.

[reference](https://www.kernel.org/doc/Documentation/gpio/sysfs.txt)

So here we are configuring the pin to be an `output` with the initial state as `high`.

Reply:
from v4.0.0
https://github.com/bluerobotics/navigator-rs/blob/0023f5c84b0a045e83595e3a2c84d0e0bca014fd/src/lib.rs#L185C1-L194C2